nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/mongodb/nxserver/config/default-ldap-users-directory-bundle.xml.nxftl

```diff
@@ -0,0 +1,351 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.directory.ldap.storage.users">
+  <#if "${nuxeo.directory.type}" == "default">
+  <!-- Using default configuration from mongodb-directories-config.xml -->
+  <#else>
+
+  <implementation class="org.nuxeo.ecm.directory.ldap.LDAPDirectoryDescriptor" />
+  <implementation class="org.nuxeo.ecm.directory.ldap.LDAPServerDescriptor" />
```

Comment:
These two `<implementation>` lines shouldn't be there I think (and nor should they be in the template this has been copied from). `<implementation>` is only used when a component is actually instantiated as a Java object.